models/topics.models.js

```diff
@@ -51,10 +52,35 @@ exports.fetchCommentsByArticleId = (article_id) => {
   const promiseArray = [];
   promiseArray.push(db.query(stringQuery, [article_id]));
   promiseArray.push(checkExists("articles", "article_id", article_id));
-  return Promise.all(promiseArray).then(([{ rows }, { exists }]) => {
+  return Promise.all(promiseArray)
+  .then(([{ rows }, { exists }]) => {
     if (rows.length === 0 && !exists) {
       return Promise.reject({ status: 404, message: "404 - Not Found" });
     }
     return rows;
   });
 };
+
+exports.insertCommentOntoArticle = (article_id, author, body) => {
+  if (typeof body !== "string") {
+    return Promise.reject({ status: 400, message: "400 - Bad Request" });
+  }
+  return checkExists("articles", "article_id", article_id)
```

Comment:
We have a get article by id that does something similar and we could use it again :)